packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -113,9 +113,10 @@ export default class EditorEnvelope extends PureComponent {
         );
     }
 
-    toggleHelmpessage = () => {
+    toggleHelpMessage = event => {
+        event.preventDefault();
         this.setState({
-            showHelpmessage: !this.state.showHelpmessage
```

Comment:
I am not sure if we should also change it with this change, but it will end more in a discussion about context. Not bug related ;)